### PR TITLE
Task2/LFU

### PR DESCRIPTION
**Pull request by sviat-h**

### CODE REVIEW

File: src/controller/AppExecutor.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package controller;
+
+import service.LFUCache;
+import service.News;
+
+public class AppExecutor {
+    private News news = new News();
+    private LFUCache cache = new LFUCache(5);
```
Comment (omilovskiy) [inline src/controller/AppExecutor.java]: what is 5?? make a constant

File: src/controller/AppExecutor.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    public void runApp() {
+        news.putToNewsList();
+
+        for (long i = 1L; i <= 10L; i++) {
```
Comment (omilovskiy) [inline src/controller/AppExecutor.java]: what is 10?? as previous
and also, why u chose long type?

File: src/controller/AppExecutor.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            cache.put(i, news.getNews("News #" + i));
+            cache.put(2L, news.getNews("News #2"));
+            cache.put(3L, news.getNews("News #3"));
+        }
+
+        for (long i = 1L; i <= 10L; i++) {
+            if (cache.get(i) != null) {
```
Comment (omilovskiy) [inline src/controller/AppExecutor.java]: contains()

File: src/service/LFUCache.java [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    private int capacity;
+
+    public LFUCache(int capacity) {
+        this.capacity = capacity;
+        this.map = new HashMap<>();
+    }
+
+    public String get(long key) {
+
+        if (map.get(key) == null) {
```
Comment (omilovskiy) [inline src/service/LFUCache.java]: contains()

File: src/service/Node.java [hunk 5]
```diff
@@ -0,0 +1,16 @@
+package service;
+
+class Node {
```
Comment (omilovskiy) [inline src/service/Node.java]: where are access modifiers?

**Review by omilovskiy — COMMENTED** [review]
Hey :) Take a look at my comments

File: src/controller/AppExecutor.java (hunk 1 above)
Comment (sviat-h) [reply src/controller/AppExecutor.java]: done

File: src/controller/AppExecutor.java (hunk 2 above)
Comment (sviat-h) [reply src/controller/AppExecutor.java]: done

File: src/controller/AppExecutor.java (hunk 3 above)
Comment (sviat-h) [reply src/controller/AppExecutor.java]: done

File: src/service/LFUCache.java (hunk 4 above)
Comment (sviat-h) [reply src/service/LFUCache.java]: done

File: src/service/Node.java (hunk 5 above)
Comment (sviat-h) [reply src/service/Node.java]: done

**Pull request merged**